### PR TITLE
WCM-626 create a blade template for rules

### PR DESCRIPTION
**Pull request by juliocamarero**

### CODE REVIEW

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/sdk/gradle-templates/src/main/resources/standalone/contenttargetingrule/src/main/resources/META-INF/resources/view.jsp [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+<liferay-theme:defineObjects />
+
+<portlet:defineObjects />
+
+<%
+Map<String, Object> context = (Map<String, Object>)request.getAttribute("context");
+%>
+
+<aui:select label="_CLASS_" name="matches" value="<%= GetterUtil.getBoolean(context.get("matches")) %>">
```
Comment (gamerson) [inline modules/sdk/gradle-templates/src/main/resources/standalone/contenttargetingrule/src/main/resources/META-INF/resources/view.jsp]: Should this be `label="_CLASS_Rule"` to make the class name in the class rule file?

Comment (juliocamarero) [inline modules/sdk/gradle-templates/src/main/resources/standalone/contenttargetingrule/src/main/resources/META-INF/resources/view.jsp]: Anything is fine, it's just a label for the select box that doesn't have any meaning...

**Comment by gamerson** [comment]
sent https://github.com/Ithildir/liferay-portal/pull/409

**Pull request closed without merging**